http/headers/accept-ch.json

```diff
@@ -93,6 +93,342 @@
             "status": {
               "experimental": true,
               "standard_track": false,
+              "deprecated": true
```

Comment:
FYI the "old" spec versions are deprecated, and either replaced with Sec- versions in new spec or just dropped in a few cases.